PrismSampleApp/PrismSampleApp/Views/HomePage.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace PrismSampleApp.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
```

Comment:
Remove this compile in each XAML cs it can be added globally in assemblyinfo.cs

Reply:
Removed this line